scripts/globals/hunts.lua

```diff
@@ -0,0 +1,1616 @@
+        [4161] = { pageId = 520, mobId = 17166669, bounty = 25, fee =   0 }, -- Tiffenotte
+        [4169] = { pageId = 521, mobId = 17170594, bounty = 15, fee =   0 }, -- Emela-ntouka
+        [4177] = { pageId = 522, mobId = 17170475, bounty = 25, fee =   0 }, -- Ratatoskr
+        [4185] = { pageId = 523, mobId = 17170499, bounty = 20, fee =   0 }, -- Kirtimukha
+        [4193] = { pageId = 524, mobId = 17170569, bounty = 35, fee =  80 }, -- Demoiselle Desolee
+        [4201] = { pageId = 525, mobId = 17174561, bounty = 25, fee =   0 }, -- Muq Shabeel
+        [4209] = { pageId = 526, mobId = 17174889, bounty = 35, fee =  75 }, -- Bloodlapper
+        [4217] = { pageId = 527, mobId = 17174682, bounty = 35, fee =  80 }, -- Hemodrosophila
+        [4225] = { pageId = 528, mobId = 17174709, bounty = 40, fee =  85 }, -- Centipedal Centruroides
+        [4233] = { pageId = 529, mobId = 17178803, bounty = 15, fee =   0 }, -- Balam-Quitz
+        [4241] = { pageId = 530, mobId = 17178885, bounty = 20, fee =   0 }, -- Hyakinthos
+        [4249] = { pageId = 531, mobId = 17178763, bounty = 25, fee =   0 }, -- Herensugue
+        [4257] = { pageId = 532, mobId = 17178689, bounty = 30, fee =   0 }, -- Coquecigrue
+        [4265] = { pageId = 533, mobId = 17448990, bounty = 35, fee =  75 }, -- Citadel Pipistrelles
+        [4273] = { pageId = 534, mobId = 17449017, bounty = 45, fee = 100 }, -- Buarainech
+        [4281] = { pageId = 535, mobId = 17449008, bounty = 45, fee = 100 }, -- Elatha
+        [4289] = { pageId = 536, mobId = 17334523, bounty = 35, fee =  80 }, -- Came-cruse
+        [4297] = { pageId = 537, mobId = 17334356, bounty = 35, fee =  80 }, -- Becut
+        [4305] = { pageId = 538, mobId = 17334482, bounty = 40, fee =  85 }, -- Grand'Goule
+        [4313] = { pageId = 539, mobId = 17334336, bounty = 45, fee = 100 }, -- Scylla
+        [4321] = { pageId = 540, mobId = 17338559, bounty = 35, fee =  80 }, -- Tikbalang
+        [4329] = { pageId = 541, mobId = 17338462, bounty = 40, fee =  85 }, -- Prince Orobas
+        [4337] = { pageId = 542, mobId = 17338386, bounty = 40, fee =  85 }, -- Graoully
+        [4345] = { pageId = 543, mobId = 17338585, bounty = 45, fee = 100 }, -- Zirnitra
+      }
+  }
+}
+
+--[[    0  |   0000000000   | 0000000000 |  0000
+      1bit |    10 bits     |  10 bits   |  4bit
+      lock |   Scyld Qty    | NM pageId #  | status
+                                          (Has distinct values) ]]--
+
+function tpz.hunts.onTrigger(player,npc,event)
+  local huntZone = player:getCharVar("[hunt]zone")
+  local huntId = player:getCharVar("[hunt]id")
+  local huntStatus = player:getCharVar("[hunt]status")
+  local scyldBits = bit.lshift(player:getCurrency("scyld"), 14)
+  local lockBit = bit.lshift(1, 24)
+  local registryZone = zone[player:getZoneID()]
+
+  -- one vana'diel day lockout timer after completing a hunt
+  if (VanadielYear() * 360) + VanadielDayOfTheYear() < player:getCharVar("[hunt]nextHunt") then
```

Comment:
Could use `if os.time() < player:getCharVar("[hunt]nextHunt") then`

Reply:
how about `if player:getCharVar("[hunt]nextHunt") < getVanaMidnight()then`

Reply:
I like this idea. Ideally we should compare against a consistent source.